### DISCUSSION TITLE
在git clone時遇到的問題&解決

### DISCUSSION THREAD

**Discussion by 11timyyy** [1101-GS4719-C/Python-Community-GS4719-C#466] [Ideas]
## 小問題
大家好，我分享一個之前在使用git clone作業0的網址時遇到了小問題!在上課時，老師有教我們使用git clone，我試了之後，電腦卻跑出這些奇妙的東西:
![image](https://user-images.githubusercontent.com/100007643/157898267-8b582634-2383-4fca-961d-f4d5d96e6564.png)
在clone之後，竟然寫Repository not found，正常的應該要跳出一個視窗才對!!

正常:
![image](https://user-images.githubusercontent.com/100007643/157898554-5460f8ac-9c27-48e9-ba3d-f61d1238675a.png)

於此經過老師的教導後，發現其實這個問題很快就可以解決喔👍👍
## 解決過程
首先，我們參考一個網頁，在搜尋欄打上:github clone remote Repository not found，跳出來的第一個搜尋結果中，找到2016年6月14日發表的這篇文章。
![image](https://user-images.githubusercontent.com/100007643/157899574-36d3fb40-fabc-452e-904b-4cd1e592e8ed.png)

點進去之後往下滑，會找到 For Mac，For Windows這邊。
![image](https://user-images.githubusercontent.com/100007643/157900259-704c168b-87c6-447e-a05b-95ec894bc547.png)

因為我是windows的，Mac的解決辦法就不是很清楚了(Sorry!)
Credential Manager其實是內建的"認證管理員"，在電腦上搜尋的到喔。那一行的意思，就是在認證管理員中，刪除相關資料。
![image](https://user-images.githubusercontent.com/100007643/157901021-d7ccad9a-3047-43f1-9564-a0340e29d7ca.png)

快完成了!接著到認證管理員中，點選Windows認證。
![image](https://user-images.githubusercontent.com/100007643/157901256-6f43fcec-fcbd-4ac7-b9c6-df694ebbe4c8.png)

接著最後一步，在"一般認證"這邊，把所有有關於"github"都移除掉。(消除在電腦上原本的記憶，留給新的使用)
![image](https://user-images.githubusercontent.com/100007643/157901834-892c6640-233b-4be6-8cd3-5d4624b84b34.png)

這樣再回去cmd git clone，就大功告成啦!
如果以後有同樣問題，不妨可以試試看!

**Comment by chia-kai-chang** [comment]
跟同學一起解決問題的感覺真好~ 😂

**Comment by yuhao112** [comment]
自己當時也有遇到這樣的問題，但助教教授都在忙，就不好意思提問了，原來當初我就是最後卡在沒有清除原本有github的資料，很感謝有你這樣的整理，也解決我的問題了!
